src/docker-osx-dev

```diff
@@ -1149,6 +1155,8 @@ function configure_includes {
 # Runs the docker-osx-dev script to to sync files.
 #
 function sync {
+  local remove_shared_folders=$1
```

Comment:
Am I missing something, or is this not used anywhere?

Also, please use `local` and `readonly` whenever possible.